web-ui/src/pages/ViewFeedbackPage.jsx

```diff
@@ -107,6 +107,36 @@ const ViewFeedbackPage = () => {
   const [dateRange, setDateRange] = useState(DateRange.THREE_MONTHS);
```

Comment:
The changes in this file are for the story "Keep context in URL: View Feedback #2268".